.env.test

```diff
@@ -1 +1,4 @@
 NODE_ENV=test
+PG_DB_NAME=gncs_test
+ASSETS_DRIVER=fake
+SESSION_DRIVER=memory
```

Comment:
für was sind die driver?

Reply:
steht in der readme :)

Reply:
https://docs.adonisjs.com/guides/testing/introduction#environment-variables